sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/common/cloud-provider/impact.adoc

```diff
@@ -0,0 +1,38 @@
+==== Data Theft
+
+The malicious users now have the ability to steal various types of data stored
+in the Cloud services, such as customer information, intellectual property,
```

Comment:
Here, cloud is written uppercase. Before, it was lowercase. Personally, I would write it in lowercase unless it is part of a name.

Reply:
```suggestion
in the cloud services, such as customer information, intellectual property,
```